bucketlist/app/templates/bucketlistitems.html

```diff
@@ -0,0 +1,78 @@
+{% extends "header.html" %}
+{% load static %}
+
+
+{% block content %}
```

Comment:
*Nitpick*: Indentation on this file is a bit off. Go over it again to see how you can improve it.